elections.tex

```diff
@@ -0,0 +1,134 @@
+\chapter{Elections}
+\label{cha:elections}
+
+\section{B-Term Officer Elections}
+\label{sec:b-term-officer-elections}
+
+Elections for all executive officers and other officers will be held every
+B-Term in two parts: executive officer elections and other officer elections
+
+\subsection{Timing}
+\label{sec:timing}
+
+Nominations for executive officer positions should be opened four weeks prior to
+the end of B-Term.
+One week later, the nominated brothers will accept or decline
+their nominations, and nominations for other officer positions should be opened.
+The following week the executive officer election will take place at a regularly
+scheduled chapter meeting, and brothers will accept or decline their nominations
+for other officer positions.
+The following week the election for other officers will take place at a
+regularly scheduled chapter meeting.
+All officers shall serve for one year, beginning the next term after their
+Election, with the exception of House Manager (see
+Item~\ref{cha:the-new-executive-board}, Section~\ref{sec:entering-office}).
```

Comment:
Since the sections automatically have the item number first, is do we need to state which item?

Reply:
I agree, but we can't change anything from the original.